Automattic/wp-calypso
# Videopress onboarding v2 : intro image

#### Proposed Changes

This PR adds a div with a background image to display something before the intro video loads.
The background image is the first frame from the video extracted with `ffmpeg -i video-onboarding-welcome-background_mp4_hd_1080p.mp4 -vf "select=eq(n\,0)" -q:v 2 videopress-onboarding-video-poster.jpeg`

### Known issues

* As we don't distribute videopress-iframe-api.js yet, I had to embed it in Calypso, but it should be changed once we have a plan for distributing the js file.

#### Testing Instructions

* Apply PR
* `yarn start`
* Go to `http://calypso.localhost:3000/setup/intro?flow=videopress`
* ✅ You should see a fixed image before the video loads
* ✅ The video should always start at the beginning, no matter where you left it off

Fix Automattic/greenhouse#1361
